### PR TITLE
Mod 2 Week2 Assessment Questions

### PR DESCRIPTION
**Pull request by memcmahon**

### CODE REVIEW

File: Mod2/Assessments/Week2Assessment.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Week 2 Assessment
+
+## Questions
+1. What does TDD stand for?
+
+1. What are three benefits of using TDD?
+
+1. Imagine you are in an interview.  The interviewer asks: How do you use TDD?
```
Comment (zoefarrell) [inline Mod2/Assessments/Week2Assessment.md:8]: I would add "How would you answer" here so it's extra clear what the student should do.

File: Mod2/Assessments/Week2Assessment.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+    Assert.Equal("Nile", dog.Name);
+}
+```
+
+1. What is a merge conflict, and when might you encounter one?
+
+1. You and a partner are working on a project together.  Your partner is working on aa-branch; you are working on bb-branch.  In as much detail as possible, describe how you both would get your work combined onto the main branch.
+
+1. Why is it important to never merge your own branch (when working on a team)?
```
Comment (zoefarrell) [inline Mod2/Assessments/Week2Assessment.md:57]: I'd prefer for us to not talk in such absolutes about never merging your own branch. All of the work flows I've used on the job involve someone else on the team approving when they think the PR looks good enough to merge. Then the owner of the PR making any last changes and merging themself. Or they put it back up for more reviews if more changes are needed.

Maybe we change the question to something like "Why is it good practice to have someone else approve and/or merge your PR?"

File: Mod2/Assessments/Week2Assessment.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
+# Week 2 Assessment
+
+## Questions
```
Comment (zoefarrell) [inline Mod2/Assessments/Week2Assessment.md:3]: In past assessments we've said how many points the questions are worth. Might be nice to add that here for clarity? Maybe 10 and 10?

File: Mod2/Assessments/Week2Assessment.md (hunk 2 above)
Comment (memcmahon) [reply Mod2/Assessments/Week2Assessment.md:57]: Yep, totally agree

**Pull request merged**